src/syft/lib/__init__.py

```diff
@@ -142,20 +145,26 @@ def _load_lib(*, lib: str, options: TypeDict[str, TypeAny] = {}) -> None:
             _regenerate_unions(lib_ast=lib_ast, client=client)
 
 
-def load(lib: str, options: TypeDict[str, TypeAny] = {}) -> None:
+def load(
+    *libs: TypeUnion[TypeList, TypeTuple, TypeSet, str],
+    options: TypeDict[str, TypeAny] = {},
+) -> None:
     """
     Load and Update Node with given library module
 
     Args:
-        lib: name of library to load and update Node with
+        *libs: names of libraries to load and update Node with (can be variadic, tuple, list, set)
         options: external requirements for loading library successfully
     """
-    try:
-        _load_lib(lib=lib, options=options)
-    except VendorLibraryImportException as e:
-        critical(e)
-    except Exception as e:
-        critical(f"Unable to load package support for: {lib}. {e}")
+    if not isinstance(libs[0], str):
```

Comment:
We shouldn't convert to string, we need to check all the arguments (even if nested) are strings.

In other words:

```
if isintance(args[0], str):
   ...
elif is_iterable(args[0]):
   # iterate through this and check if the components are strings and load them
else:
   # error

Reply:
Done.